src/server/implementation/cmd/forward.c

```diff
@@ -8,8 +8,19 @@
 #include <stdio.h>
 #include "server_implementation.h"
 
-int forward(__attribute__ ((unused)) data_t *data,
-    __attribute__ ((unused)) char **args)
+int forward(data_t *aData, __attribute__ ((unused)) char **aArgs)
```

Comment:
the UwU norm is not applied on the server because there already is the Epitech norm, so just call id "data" and "args"